### ISSUE TITLE
Visual feedback of progress of the large .sql file ingestion during site update

### ISSUE THREAD

**Issue by jteresco** [TravelMapping/DataProcessing#112]
One of the frustrations with the problem mentioned in #111 is that we don't necessarily know that the mysql process is or is not making progress.  A successful ingestion runs for several minutes.

Possible enhancement: see the "pipe viewer" tool shown in the accepted answer here: https://dba.stackexchange.com/questions/17367/how-can-i-monitor-the-progress-of-an-import-of-a-large-sql-file

**Comment by michihdeu** [comment]
I think that the site is usually fully working when the "red update in progress message" is displayed. Only for some seconds (I guess when the DB is exchanged) some data is not displyed, e.g. the overall system info on the start page. Is it really "several minutes"? Maybe we should only display the message when the critical part is being processed?

**Comment by jteresco** [comment]
This wouldn't be displayed to users, just to the person running the site update process who is wondering whether the DB is just taking a long time to load but still making progress, or if it's stuck with the the situation in #111.

**Comment by jteresco** [comment]
Anyway, I leave the red banner up the whole time the primary DB is being updated, but I agree that's probably not necessary any longer.

**Comment by yakra** [comment]
"Visual feedback" = a series of `.....` similar to what we see during `read_wpts_for_highway_system`, etc.?

**Comment by jteresco** [comment]
> "Visual feedback" = a series of `.....` similar to what we see during `read_wpts_for_highway_system`, etc.?

Yes, or whatever that "pipe viewer" does, assuming it is an appropriate tool for this.  One hesitation in doing so would be that it could add a new software dependency.  So I think we'd want it done inside a conditional that uses the viewer tool if it's available, do the existing command if not.
